sd/etcdv3/doc.go

```diff
@@ -0,0 +1 @@
+package etcdv3
```

Comment:
One or two sentences of doc comment would be appreciated!

Reply:
I copied and updated the doc comment from sd/etcd to sd/etcdv3 for consistency sake.